ProcessSlave.php

```diff
@@ -389,9 +389,9 @@ protected function prepareEnvironment(\React\Http\Request $request)
             $_SERVER['HTTP_' . strtoupper(str_replace('-', '_', $name))] = $value;
         }
 
-        //HTTP_X_REAL_IP is always set, either through ProcessMaster or by a proxy
-        //in front of ProcessMaster.
-        $_SERVER['REMOTE_ADDR'] = $_SERVER['HTTP_X_REAL_IP'];
+        //Using ProcessManager in front of the slaves masks the real/proxy ip
+        $_SERVER['REMOTE_ADDR'] = $_SERVER['HTTP_X_REMOTE_IP'];
```

Comment:
@staabm yes, that's the main reason of us having to do that. It works well with Symfony in a reverse proxy environment now ;)

@marcj yes, I think that's a better name.